include/cvodes/cvodes.h

```diff
@@ -284,8 +284,9 @@ SUNDIALS_EXPORT int CVodeGetNumStabLimOrderReds(void* cvode_mem,
 SUNDIALS_EXPORT int CVodeGetActualInitStep(void* cvode_mem, sunrealtype* hinused);
 SUNDIALS_EXPORT int CVodeGetLastStep(void* cvode_mem, sunrealtype* hlast);
 SUNDIALS_EXPORT int CVodeGetCurrentStep(void* cvode_mem, sunrealtype* hcur);
-SUNDIALS_EXPORT int CVodeGetCurrentState(void* cvode_mem, N_Vector* y);
-SUNDIALS_EXPORT int CVodeGetCurrentStateSens(void* cvode_mem, N_Vector** yS);
+SUNDIALS_EXPORT int CVodeGetCurrentState(void* cvode_mem, N_Vector* y_1d);
```

Comment:
`y` is an output not an array
```suggestion
SUNDIALS_EXPORT int CVodeGetCurrentState(void* cvode_mem, N_Vector* y);
```